rules/do-you-tie-knowledge-to-the-role/rule.md

```diff
@@ -12,9 +12,9 @@ guid: bbe468cd-c3d5-4d67-86ba-22f4918a1acb
 
```

Comment:
The redirects is missing when you [rename a rule](https://github.com/SSWConsulting/SSW.Rules.Content/wiki/How-to-Rename-Rules)

```suggestion
redirects:
  - do-you-keep-the-knowledge-in-the-cloud
```